packages/bpk-component-pagination/src/BpkPaginationPage.module.scss

```diff
@@ -18,11 +18,13 @@
 
 @import '~bpk-mixins/index';
 
+$bpk-spacing-v2: true;
+
 .bpk-pagination-page {
-  min-width: $bpk-spacing-xl;
-  height: $bpk-spacing-xl;
+  min-width: bpk-spacing-xxl();
+  height: bpk-spacing-xxl();
   padding: 0;
-  border-radius: $bpk-border-radius-pill;
+  border-radius: $bpk-border-radius-pill-lg;
```

Comment:
Should we just use 50% instead? The intent here is to have a circle and while `border-radius: $bpk-border-radius-pill-lg;` accidentally achieves this it seems more future proof to just use 50%.

Reply:
It seems like the tokens are used somewhat more frequently in our code rather than percentages so it seemed like a better choice in terms of consistency. Happy to use 50% if that's safer and we know it will most likely always remain a circle

Reply:
You're definitely correct, but I feel like 50% better expresses the intent here. Maybe we can see what @olliecurtis and @georgegillams thinks

Reply:
That's definitely not what we want...

![Screenshot 2021-08-23 at 18 25 22](https://user-images.githubusercontent.com/30267516/130490480-63331b61-aeb0-45a0-8dc3-7a7dea0152ce.png)

Reply:
Oh this is for the pagination ones. Yeah as they're circular that works 👍

Reply:
I agree with @k0nserv that `50%` expresses the intent more accurately.

Reply:
Makes sense 👍 Thanks both!